core/templates/dev/head/css/oppia.css

```diff
@@ -3926,12 +3926,21 @@ md-card.preview-conversation-skin-supplemental-card {
   position: absolute;
   word-break: break-all;
 }
-
 .oppia-activity-summary-tile .objective {
-  height: 60px;
+  display: -webkit-box;
   font-family: "Rubik", "Roboto", Arial, sans-serif;
   font-size: 0.85em;
+  height: 70px;
+  line-height: 1.26;
+  /*`max-height` is calculated as: 1.26 (line-height) * 3 (lines) / 0.85
+  (font-size). An additional (1.26 (line-height) * 0.85 (font-size) * 0.25) is added
+  toaccount for letters (like y,jand g) that fall beneath the baseline. */
```

Comment:
@seanlip I've added  `max-height` as a fallback for non-webkit browsers, for which
 `-webkit-line-clamp` won't work. This limits the objective to three lines. Also, it didn't affect the overall height of the card.